runtime/plugin/tohtml.lua

```diff
@@ -0,0 +1,58 @@
+if vim.g.loaded_2html_plugin then
+  return
+end
+vim.g.loaded_2html_plugin = 'nvim10.0_v1'
```

Comment:
Why this specific value? It should just be 0 or 1 (**if** we want to keep legacy conventions for Lua modules, which I'm not convinced about).

(Note that these are just general guidances; I know this PR is still draft, and if you're incrementally converting from the legacy plugin, this is OK. I just want you to be clear as early as possible about our expectations for the final product. And to be perfectly: it is **not** a 1:1 translation of the old plugin just in Lua; it should have a much smaller API surface and be focused on the goal of providing a printable format via browsers.)

Reply:
It was that value because `TOhtml` used that format.

I have updated and set it to `true` (because `plugin/man.lua` does it that way)

Also, I have removed the options that remove specific decorations (like conceal).

Reply:
Right; I just want to make it clear that we don't consider `TOhtml` to be a good model for... basically, anything ;)

We'd love to have the basic functionality with none of the legacy baggage. (Which should make your job easier as well, since you can just skip any of the more byzantine parts of the code...)

Reply:
So, should multi-window diffs be supported? (That's basically the last major feature thing that I'm planning to add)

Reply:
Certainly not needed for a first PR. (In fact, I would have stopped to take stock much earlier...)

Reply:
All this makes it sound like I want to discourage you -- I don't, we really appreciate your efforts!